packages/components/src/components/modal/modal.jsx

```diff
@@ -80,6 +80,17 @@ const ModalElement = ({
         };
     }, []);
 
+    React.useEffect(() => {
+        const closeOnEscButton = e => {
+            if (e.key === 'Escape') {
+                toggleModal?.();
+            }
+        };
+        window.addEventListener('keydown', closeOnEscButton);
+        return () => window.removeEventListener('keydown', closeOnEscButton);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we handle the dependancies instead of ignoring them?

Reply:
this line (// eslint-disable-next-line react-hooks/exhaustive-deps) was redundant, so I will delete it